var/spack/repos/builtin/packages/pfunit/package.py

```diff
@@ -93,6 +93,14 @@ class Pfunit(CMakePackage):
     variant("esmf", default=False, description="Enable esmf support")
     variant("docs", default=False, description="Build docs")
 
+    # pFUnit brought in mpi_f08 support in 4.4.0
```

Comment:
You might want to add a `conflicts("+mpi_f08", when="~mpi")` around line 146 so these get flagged at concretization time:

```
$ spack spec pfunit+mpi_f08~mpi
==> Error: failed to concretize `pfunit~mpi+mpi_f08` for the following reasons:
     1. 'pfunit' required multiple values for single-valued variant 'mpi'
     2. pfunit: '+mpi_f08' conflicts with '~mpi'
     3. pfunit: '+mpi_f08' conflicts with '~mpi'
        required because conflict is triggered when +mpi_f08 
          required because pfunit~mpi+mpi_f08 requested explicitly 
        required because conflict constraint ~mpi 
          required because pfunit~mpi+mpi_f08 requested explicitly 
```

Reply:
Ahhh. Smart. I'll add that.